### ISSUE TITLE
UTF-8 issue

### ISSUE THREAD

**Issue by LienReyserhove** [trias-project/daisie-checklist#9]
I have the same problem here with the UTF-8 encoding of the taxon core.
The encoding of the raw data file is ok, but when exporting it to a .csv and also when viewing it in R, it renders some problems with the encoding. I guess this is the same issue as [this issue](https://github.com/trias-project/alien-plants-belgium/issues/41) for the manual of alien plants. I cannot remember if we ever solved this one, although problems with the encoding did not occur for a long time..
I checked for the manual of alien plants, and the UTF-8 issue remains there as well, so I guess we didn't solve the problem yet.
@peterdesmet can you fresh up my mind please?

**Comment by peterdesmet** [comment]
I don't think we solved that issue. I would suggest to ignore it for now (leaving this issue open) and for the final files, to just run it on a Mac or Linux.

**Comment by LienReyserhove** [comment]
That's what I thougth.
Ok, the thing is that it makes it very difficult for me to scan the scientific names for error (groups). But as we are talking about 12000 taxa here, I should probably be not too difficult with the scientific names.

**Comment by peterdesmet** [comment]
Indeed, would start data cleaning those.

**Comment by peterdesmet** [comment]
Leaving issue open because data still needs to be run on Mac to solve this.
